### ISSUE TITLE
Foreground of `MainActivity` is transparent: background (splash) leaks through it on light themes

### ISSUE THREAD

**Issue by erikhuizinga** [vector-im/element-android#2927]
**Describe the bug**

`MainActivity` / `FragmentLoadingBinding` / `fragment_loading.xml` has a transparent foreground if the default light theme is used. The `windowBackground` is visible through the foreground. This is not an issue in the dark theme.

I'm not sure how big of an issue this is, because usually the main activity is shown so briefly that it is not visible. However, I'm not aware of cases where that activity may be visible for a longer time; then this bug is visible to users!

**To Reproduce**

0. Wipe app data / uninstall Element
1. Use a device that uses the default launcher theme, i.e. a light theme, or use an older Android API level that doesn't support dark system themes
2. Disable this line: https://github.com/vector-im/element-android/blob/v1.1.0/vector/src/main/java/im/vector/app/features/MainActivity.kt#L112 so that the main activity stays visible (this helps see the bug)
3. Run the app, I used a debug build (if that matters)
4. See that the `fragment_loading` layout is shown over `splash.xml` which the default theme's `windowBackground`.

<details>
<summary>Diff applicable to step 2 above</summary>=

```diff
diff --git a/vector/src/main/java/im/vector/app/features/MainActivity.kt b/vector/src/main/java/im/vector/app/features/MainActivity.kt
index 351163b02..5e0198788 100644
--- a/vector/src/main/java/im/vector/app/features/MainActivity.kt
+++ b/vector/src/main/java/im/vector/app/features/MainActivity.kt
@@ -109,7 +109,7 @@ class MainActivity : VectorBaseActivity<FragmentLoadingBinding>(), UnlockedActiv
         if (args.clearCache || args.clearCredentials) {
             doCleanUp()
         } else {
-            startNextActivityAndFinish()
+            // startNextActivityAndFinish()
         }
     }
```

</details>

**Expected behavior**

I expect that the splash window background doesn't leak through the foreground layout anywhere in the app.

Possible solutions:

- Set an opaque background in fragment_loading.xml, possibly only for the specific affected API levels.
- Apply a default background in the default theme, just like in the dark theme (because it doesn't have this issue).

If desirable, I can make a contribution. Please let me know if I should pick this up.

**Screenshots**

| Light theme (default) | Dark theme |
|-|-|
| ![image](https://user-images.githubusercontent.com/19374736/109310635-14aeab00-7845-11eb-827e-7c7568690dc6.png) | ![image](https://user-images.githubusercontent.com/19374736/109310647-18423200-7845-11eb-8268-87545a0782b9.png)|
| Note that the white element logo is shown behind the green element logo in the foreground | no issue |

**Smartphone (please complete the following information):**

 - Device: emulated Pixel 3a
 - OS: emulated Android 11 (API 30)

**Additional context**

 - App version and store v1.1.0 `./gradlew assembleGplayDebug`
 - Homeserver: N/A, clean install without authentication

**Comment by bmarty** [comment]
Thanks for creating this issue. This is actually a regression when I did the migration to ViewBindings. I will fix it for the next release (v1.1.1)

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
http://user-images.githubusercontent.com/19374736/109310647-18423200-7845-11eb-8268-87545a0782b9.png is demonstrating that the splash screen for Element has been dark. Thus, is http://github.com/vector-im/element-android/issues/3550 one regression of this?
